Makefile

```
@@ -18,6 +18,7 @@ lint:
 test: lint
 	@echo "Running test suite..."
 	@cd test && make
+	@cd adjoint && make
```

Does it not need to be this?
```suggestion
	@cd test/adjoint && make
```
I thought Make returned to the previous directory after each line?

---

Sorry, you're right! Commited the suggestion